package.json

```diff
@@ -45,7 +45,7 @@
     "coveralls": "^2.11.4",
     "del": "^1.2.1",
     "ethereumjs-abi": "0.6.4",
-    "ethereumjs-stub-rpc-server": "1.0.0",
+    "ethereumjs-stub-rpc-server": "2.0.5",
```

Comment:
There is a bug in the stub server that caused the tests to fail with this change.